apps/server/src/modules/files-storage/uc/files-storage.uc.spec.ts

```diff
@@ -45,91 +40,85 @@ function createObservable<T = unknown>(data: T, headers: AxiosResponseHeaders =
 }
 
 const getFileRecordsWithParams = () => {
-	const userId1 = new ObjectId().toHexString();
+	const userId = new ObjectId().toHexString();
 	const schoolId = new ObjectId().toHexString();
 
-	const fileRecords1 = [
-		fileRecordFactory.buildWithId({ parentId: userId1, schoolId, name: 'text.txt' }),
-		fileRecordFactory.buildWithId({ parentId: userId1, schoolId, name: 'text-two.txt' }),
-		fileRecordFactory.buildWithId({ parentId: userId1, schoolId, name: 'text-tree.txt' }),
+	const fileRecords = [
+		fileRecordFactory.buildWithId({ parentId: userId, schoolId, name: 'text.txt' }),
+		fileRecordFactory.buildWithId({ parentId: userId, schoolId, name: 'text-two.txt' }),
+		fileRecordFactory.buildWithId({ parentId: userId, schoolId, name: 'text-tree.txt' }),
 	];
 
-	const params1: FileRecordParams = {
+	const params: FileRecordParams = {
 		schoolId,
-		parentId: userId1,
+		parentId: userId,
 		parentType: FileRecordParentType.User,
 	};
 
-	return { params1, fileRecords1, userId1 };
+	return { params, fileRecords, userId };
 };
 
 const getFileRecordWithParams = () => {
-	const userId1 = new ObjectId().toHexString();
+	const userId = new ObjectId().toHexString();
 	const schoolId = new ObjectId().toHexString();
 
-	const fileRecord1 = fileRecordFactory.buildWithId({ parentId: userId1, schoolId, name: 'text.txt' });
+	const fileRecord = fileRecordFactory.buildWithId({ parentId: userId, schoolId, name: 'text.txt' });
 
-	const params1: SingleFileParams = {
-		fileRecordId: FileRecordParentType.User,
+	const params: SingleFileParams = {
+		fileRecordId: fileRecord.id,
 	};
 
-	return { params1, fileRecord1, userId1 };
+	return { params, fileRecord, userId };
+};
+
+const getRequest = () => {
+	return createMock<Request>({
```

Comment:
type of the return to function and 
const axiosRequest: XXXAxiosRequestTypeXXX = createMock

return axiosRequest;

Reply:
Removed direct return. But type comes frome express and not axios.